dist/pkg/PKGBUILD.in

```diff
@@ -0,0 +1,32 @@
+# Maintainer: Christopher Hoage <iam@chrishoage.com>
+
+pkgname=ipmi-fan-control
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="SuperMicro IPMI fan control daemon"
+url="https://github.com/chenxiaolong/$pkgname"
+source=("@TARBALL_NAME@" "ipmi-fan-control.service")
+backup=("etc/$pkgname.toml")
+arch=("x86_64")
+license=("GPLv3+")
+makedepends=("cargo")
+optdepends=("ipmitool" "smartmontools")
```

Comment:
I'd suggest putting `ipmitool` in `depends=(...)` since ipmi-fan-control can't run without it.